docs/settings_ref.rst

```diff
@@ -180,6 +180,24 @@ Allows you to set a leeway of the JWT token. See the official
 `PyJWT <https://pyjwt.readthedocs.io/en/stable/usage.html>`__ docs for more information.
 
 
+FAILED_RESPONSE_FUNCTION
```

Comment:
Could we rename this to `CUSTOM_FAILED_RESPONSE_VIEW` (or similar)? I want it to be clear that it's not a required command, and that it's intended to be a function based view.

Reply:
Changed to `CUSTOM_FAILED_RESPONSE_VIEW`.